aws/rds.go

```diff
@@ -0,0 +1,95 @@
+package aws
+
+import (
+	"time"
+
+	awsgo "github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/rds"
+	"github.com/gruntwork-io/cloud-nuke/logging"
+	"github.com/gruntwork-io/gruntwork-cli/errors"
+)
+
+func waitUntilRdsDeleted(svc *rds.RDS, input *rds.DescribeDBInstancesInput) error {
+	for i := 0; i < 240; i++ {
+		_, err := svc.DescribeDBInstances(input)
+		if err != nil {
+			if awsErr, isAwsErr := err.(awserr.Error); isAwsErr && awsErr.Code() == "DBInstanceNotFound" {
+				return nil
+			}
+
+			return err
+		}
+
+		time.Sleep(1 * time.Second)
+		logging.Logger.Debug("Waiting for RDS to be deleted")
+	}
+
+	return RdsDeleteError{}
+}
+
+func getAllRdsInstances(session *session.Session, region string, excludeAfter time.Time) ([]*string, error) {
+	svc := rds.New(session)
+
+	result, err := svc.DescribeDBInstances(&rds.DescribeDBInstancesInput{})
+
+	if err != nil {
+		return nil, errors.WithStackTrace(err)
+	}
+
+	var names []*string
+
+	for _, database := range result.DBInstances {
+		//if excludeAfter.After(*database.InstanceCreateTime) {
+		names = append(names, database.DBInstanceIdentifier)
+		//}
+	}
+
+	return names, nil
+}
+
+func nukeAllRdsInstances(session *session.Session, names []*string) error {
+	svc := rds.New(session)
+
+	if len(names) == 0 {
+		logging.Logger.Infof("No Relational Database Service to nuke in region %s", *session.Config.Region)
+		return nil
+	}
+
+	logging.Logger.Infof("Deleting all Relational Database Services in region %s", *session.Config.Region)
+	var deletedNames []*string
```

Comment:
NIT: Go idiom is to typically initialize the variable inline with the declaration. That way, it is more explicit what is the initial value. You typically only declare variables if you use `if` blocks to initialize.

```suggestion
	deletedNames := []*string{}
```